### ISSUE TITLE
Inconsistent Hop behavior on folds

### ISSUE THREAD

**Issue by mystilleef** [phaazon/hop.nvim#124]
`Hop` doesn't account for folds in a consistent manner. In my testing, `Hop` does not annotate folds when `foldmethod=indent`. However, it annotates folds when `foldmethod=syntax`.

It's best if `Hop` just annotated all folds regardless of the fold method. I've attached a video showing the issue. I changed `foldmethod` from `syntax` to `indent` and activate `HopWord`. You'll see the inconsistent annotation behavior.

**Comment by phaazon** [comment]
You are right, folds are not well supported by Hop (and no code currently tries to do so). Efforts are clearly needed to make them supported correctly. Thanks for reporting.

**Comment by mystilleef** [comment]
For the benefit of users who want to jump to folds, `HopLine` works consistently for all folds.

**Comment by phaazon** [comment]
#222 should have fixed that.
